graphql_compiler/tests/test_helpers.py

```diff
@@ -389,6 +389,23 @@ def get_test_macro_registry():
                 }
             }
         }''', {}),
+        # The same as out_AnimalRichYoungerSiblings, but with a filter after the target.
+        ('''{
+            Animal @macro_edge_definition(name: "out_Animal_RichYoungerSiblings_2") {
```

Comment:
Oh dang, that's a wild one... I never even thought of this.

@lodrion @eyurtsev check out this macro edge definition.